.github/container/Dockerfile.base

```diff
@@ -54,6 +54,7 @@ apt_packages=(
   liblzma-dev
   python-is-python3
   python3-pip
+  python3-venv
```

Comment:
This could conflict with pip-installed venv packages. Can we use pip to install it? The rule of thumb here is to install with pip as much as possible and avoid distro-repos.

Reply:
OK, I switched to `virtualenv` both for the parts that run inside the container and the script (`install.sh`) that the user runs on their machine.